tools/blocks-crawler/src/main.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0
+
+use blocks::{fetch_block_info, parse_block_json, retrieve_block_json, write_json};
+use std::fs::create_dir_all;
+#[macro_use]
+extern crate log;
+
+mod blocks;
+
+#[tokio::main]
+async fn main() {
+    env_logger::init();
+    // Create output folders
+    create_dir_all(&"output/succeeded/").unwrap();
+    create_dir_all(&"output/failed/").unwrap();
+
+    // Fetch n blocks info from graphql api
```

Comment:
I think n should be configurable, maybe via CLI args

Reply:
fixed